apps/api/v2/src/ee/event-types/inputs/create-event-type.input.ts

```diff
@@ -41,6 +41,10 @@ export class CreateEventTypeInput {
   @IsArray()
   locations?: EventTypeLocation[];
 
+  @IsBoolean()
+  @IsOptional()
+  disableGuests?: boolean;
```

Comment:
I see that `EventTypesService.createUserEventType` invoked by schedule create endpoint calls `createEventType` from platform libraries that is `packages/trpc/server/routers/viewer/eventTypes/create.handler.ts`.

I checked it's schema `createEventTypeInput` and there is no `disableGuests` - it works because in the handler we have `...rest` that ends up in create body.

So we should also update schema `packages/prisma/zod/custom/eventtype.ts` by adding `disableGuests` boolean and same for update event type i guess.